src/components/link/daikin-link.visual.test.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  type InferStorybookArgTypes,
+  clipFor,
+  describeEach,
+  getStorybookIframeURL,
+} from "#tests/visual";
+import { expect, test } from "@playwright/test";
+import type { DAIKIN_LINK_ARG_TYPES } from "./stories/common";
+
+type StoryArgs = InferStorybookArgTypes<typeof DAIKIN_LINK_ARG_TYPES>;
+
+const getPageURL = (args: StoryArgs = {}) =>
+  getStorybookIframeURL("components-link--default", args);
+
+describeEach(["normal"] as const, (showVisited: "normal" | "showVisited") => {
+  // NOTE: Has not implemented the VRT for `showVisited`
```

Comment:
I don't think we need this `describeEach`.

```suggestion
// FIXME: VRT is not implemented for `:visited` styles, as it will require `launchPersistentContext` which cannot be done with containers.
```